### PR TITLE
[MUI-537]: Create StatusInfo Component

### PR DESCRIPTION
**Pull request by mr-teddy-bear**
# Description

Create StatusInfo Component

## Jira ticket(optional)

[MUI-537](https://minterestfinance.atlassian.net/jira/software/projects/MUI/boards/7?assignee=62591503eee0a9006ab98b74&selectedIssue=MUI-537)

## Type of change

Please pick options that are relevant.

- [ ] Bug fix OR refactoring (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature or refactoring that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/components/StatusInfo/StatusInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      case 'active':
+        return 'rgba(0, 207, 83, 0.1)';
+      case 'expired':
+        return 'rgba(243, 91, 83, 0.1)';
+      case 'removed':
+        return 'rgba(20, 110, 255, 0.1)';
+    }
+  }, [variant]);
+
+  return <Wrapper bgColor={bgColor}>{variant}</Wrapper>;
```
Comment (gavrilin-remote) [inline src/components/StatusInfo/StatusInfo.tsx:36]: Wrap `variant` in Typography. This way we can control text style via typography.

File: src/components/StatusInfo/StatusInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  lineHeight: '21px',
+  color: '#061953',
+  background: bgColor,
+  textTransform: 'capitalize',
+  borderRadius: '4px',
+}));
+
+export const StatusInfo: FC<Props> = ({ variant }: Props) => {
+  const bgColor = useMemo(() => {
+    switch (variant) {
```
Comment (gavrilin-remote) [inline src/components/StatusInfo/StatusInfo.tsx:26]: switch can be transformed into object: Record<variant, string>, but up to you

**Review by p1geon — APPROVED** [review]
LGTM, except for a typography mentioned by Oleg.

File: src/components/StatusInfo/StatusInfo.tsx (hunk 2 above)
Comment (mr-teddy-bear) [reply src/components/StatusInfo/StatusInfo.tsx]: Done

File: src/components/StatusInfo/StatusInfo.tsx (hunk 1 above)
Comment (mr-teddy-bear) [reply src/components/StatusInfo/StatusInfo.tsx]: Done

**Review by gavrilin-remote — APPROVED** [review]

**Pull request merged**